### PR TITLE
Feature/cab-3739

### PR DESCRIPTION
**Pull request by huiaic**
scripts to run load tests for bulk edit

### CODE REVIEW

**Review by jfshenuw — APPROVED** [review]
You might consider using "const" and "let" in place of "var".

File: content-api-crud/scripts/writeBulkEditBody.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// JSR223 script used internally by search-and-bulk-edit.yml
+// Reads list of IDs, Revision Ids and writes the request body to be used by
+// content-api bulk edit.
+
+var idList = new java.util.ArrayList(
```
Comment (farmas) [inline content-api-crud/scripts/writeBulkEditBody.js:5]: This is some interop from javascript to Java? For my own understanding, does the tool use Node? (does it have access to Node's 'fs', for example).

File: distributed-testing/install-bzt-jmeter.yml [hunk 2]
```diff
@@ -15,8 +15,9 @@ settings:
   artifacts-dir: ./tmp
 
 modules:
+  # https://gettaurus.org/docs/JMeter/
   jmeter:
-    properties:
-      hostname: test
+    force-ctg: true # required for higher concurrency thread groups
+    detect-plugins: true
```
Comment (farmas) [inline distributed-testing/install-bzt-jmeter.yml:21]: What are the plugins that are being detected?

**Review by farmas — APPROVED** [review]

File: content-api-crud/scripts/writeBulkEditBody.js (hunk 1 above)
Comment (huiaic) [reply content-api-crud/scripts/writeBulkEditBody.js:5]: It does not use Node. 
 https://stackoverflow.com/questions/45668451/jmeter-jsr-223-language-js-vs-javascript

File: distributed-testing/install-bzt-jmeter.yml (hunk 2 above)
Comment (huiaic) [reply distributed-testing/install-bzt-jmeter.yml:21]: Jmeter Plugins. When true, it's supposed to automatically detect what plugins to install

**Pull request merged**